lib/virthck.rb

```diff
@@ -124,4 +124,9 @@ def client_alive?(name)
     s_id = @id.to_s.rjust(4, '0')
     `ps -A -o cmd | grep '[\-]name HCK-Client#{id}_#{s_id}'`.split("\n").any?
   end
+
+  def studio_alive?
+    s_id = @id.to_s.rjust(4, '0')
+    `ps -A -o cmd | grep '[\-]name HCK-Studio_#{s_id}'`.split("\n").any?
```

Comment:
Maybe we can create a base method for checking live machines and extend it for client_alive? and studio_alive?